compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -603,9 +603,8 @@ void addCPULinalgExtTileAndVectorizePipeline(
   addTileAndDistributePasses(funcPassManager);
   funcPassManager.addPass(
       createLLVMCPUTilePass(tilingConfig.getVectorCommonParallelLevel()));
-  // TODO: Should only apply decomposition here?
-  funcPassManager.addPass(
-      IREE::LinalgExt::createTileAndDecomposeAttentionPass());
+  funcPassManager.addPass(IREE::LinalgExt::createTileAttentionPass());
```

Comment:
Can we add a TODO and explain why we have this separate pass? Perhaps something like:

```
TODO: Remove the pass once we have PartialOpTilingInterface implemented.
```